proto/streamlit/proto/ForwardMsg.proto

```diff
@@ -86,13 +85,11 @@ message ForwardMsgMetadata {
   // and a client that receives it should do the same.
   bool cacheable = 1;
 
-  // Each delta applies to a particular ID. Only set for Delta messages.
-  uint32 delta_id = 2;
+  // The path that identifies a delta's location in the report tree.
+  // Only set for Delta messages.
+  repeated uint32 delta_path = 2;
```

Comment:
Yeah - the way that our ForwardMsgCaching works is that we cache the entirety of the Delta _without_ the metadata. So anything that we don't want to be part of an element's identity, for caching purposes, can't live inside it. (So: just as you say.)

Reply:
Fair enough!  In that case, let's add a comment here explaining exactly what you just said (why we stick in things to ForwardMsgMetadata rather than the more apparently correct locations) =)